hi-ml-histopathology/src/histopathology/configs/classification/BaseMIL.py

```diff
@@ -93,6 +88,27 @@ def setup(self) -> None:
         if not self.is_finetune:
             self.encoder.eval()
 
+    def download_ssl_checkpoint(self, run_id: str) -> CheckpointDownloader:
+        downloader = CheckpointDownloader(
+            aml_workspace=get_workspace(),
+            run_id=run_id,
+            checkpoint_filename="last.ckpt",
+            download_dir="outputs/",
+            remote_checkpoint_dir=Path(self.checkpoint_folder_path)
+        )
+        os.chdir(fixed_paths.repository_root_directory().parent)
```

Comment:
While you're modifying that, can we get rid of this os.chdir? Changing working directory is brittle and error prone. Downloads can go into `self.outputs_folder` instead